extension/js/background_page/migrations.ts

```diff
@@ -2,6 +2,29 @@
 
 'use strict';
 
+import { KeyInfo, KeyUtil } from '../common/core/crypto/key.js';
+import { Catch } from '../common/platform/catch.js';
+import { GlobalStore } from '../common/platform/store/global-store.js';
+import { KeyStore } from '../common/platform/store/key-store.js';
+
+const addKeyInfoFingerprints = async () => {
+  for (const acctEmail of await GlobalStore.acctEmailsGet()) {
+    const keyinfos = await KeyStore.get(acctEmail);
+    const output: KeyInfo[] = [];
+    for (const keyinfo of keyinfos) {
+      const processed = await Catch.undefinedOnException(KeyUtil.keyInfoObj(await KeyUtil.parse(keyinfo.private)));
```

Comment:
This is a difficult decision to make - let it fail or not.

I tend to lean towards fail quickly and fail loud. Therefore we should not be ignoring errors here.

However the "fail loud" part is difficult, since this happens in a background script.

Either way, let's make it fail if there is an error.